cmd/_integration-tests/transport/handlers/handlers.go

```diff
@@ -20,6 +21,14 @@ type transportpermutationsService struct{}
 
 // GetWithQuery implements Service.
 func (s transportpermutationsService) GetWithQuery(ctx context.Context, in *pb.GetWithQueryRequest) (*pb.GetWithQueryResponse, error) {
+	spew.Dump(ctx)
+	if rurl := ctx.Value("request-url").(string); rurl != "/getwithquery" {
+		panic("Context Value: request-url, expected 'GetWithQuery' got " + rurl)
```

Comment:
The string in this panic does not match the value that's actually being tested against. It's an integration test, so it's not a critical path anything, but it's an inconsistency nonetheless.